src/node-explorer-provider.ts

```diff
@@ -83,7 +83,7 @@ export class NodeExplorerProvider
         rootDir = '~';
         dirDesc = '~';
       }
-      const uri = vscode.Uri.parse(`ts://${element.TailnetName}/${element.HostName}/${rootDir}`);
+      const uri = vscode.Uri.parse(`ts://${element.TailnetName}/${element.HostName}${rootDir}`);
```

Comment:
```suggestion
      const uri = vscode.Uri.parse(
        `ts://${element.TailnetName}/${element.HostName}/${
          rootDir.startsWith('/') ? rootDir.slice(1) : rootDir
        }`
      );
```

This created an issue when the `rootDir` does not have a leading slash, like with `~`. Not a great solution, but it will work for now. I have a branch almost ready that does a better job at standardizing utilities for managing Uri's.

Reply:
I think we're overcomplicating it. I'll post an alternative.